package.json

```diff
@@ -29,64 +29,64 @@
     "e2e:open": "cypress open"
   },
   "dependencies": {
-    "@aeternity/aepp-sdk": "13.3.3",
+    "@aeternity/aepp-sdk": "14.0.0",
```

Comment:
don't forget related changes from #959 😉

Reply:
Oh yeah I overlooked that. Thanks, fixed